webpack-intro/package.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Looks like you copied the main `package.json` file to create this one. In the future just use `npm init -y` to create a new `package.json` file.